docs/en/ingest-management/fleet/migrate-elastic-agent.asciidoc

```diff
@@ -115,6 +115,38 @@ Using that value for `deployment-id`, the new {fleet-server} host URL is:
 [role="screenshot"]
 image::images/migrate-agent-fleet-server-host.png[Fleet server hosts showing the new host URL]
 
+[discrete]
+[[migrate-elastic-agent-reset-policy]]
+=== Reset the {ecloud} policy
+
+On your target cluster, certain settings from the original {agent} policies may still be retained, and need to be updated to reference the new cluster.
```

Comment:
Should we add the apm url in the apm integration policy as an example? I think it is a bit vague this way.

Reply:
Yes it is a bit confusing. Basically the command resets the elastic cloud agent policy, but this means that any integration policies installed to it are also reset.  
I'd just like to highlight somewhere the specific case of apm url not being refreshed, so users can find it easily.

Reply:
Fixed. :-)